src/HereAuth/Command/HereAuthCommand.php

```diff
@@ -38,7 +39,10 @@ public function __construct(HereAuth $main, $name, $desc, $usage, ...$aliases){
 		parent::__construct($name, $desc, $usage, $aliases);
 	}
 
-	public function getMain():HereAuth{
+	/**
+	 * @return HereAuth
+	 */
+	public function getMain(){
```

Comment:
Sorry i fixed this a little bit ago, but the branch i was using was the pw-hash branch so i tried pushing it onto this, ima restart